tests/foreman/longrun/test_puppet_upgrade.py

```diff
@@ -2,11 +2,11 @@
 
 :Requirement: Puppet
 
-:CaseAutomation: Automated
+:CaseAutomation: NotAutomated
 
-:CaseLevel: Acceptance
+:CaseLevel: System
 
-:CaseComponent: CLI
+:CaseComponent: ConfigurationManagement
```

Comment:
I think the more appropriate CaseComponent is `Puppet` here as per Bugzilla component ? what do you think ?

Reply:
I'd say both are equally good. `Puppet` has description "Bugs around Puppet Integration", `ConfigurationManagement` has description "Bugs related to Puppet functionality".

I thought ConfigurationManagement will be slightly better match, but since all these tests are `@stubbed`, I don't think it really matters. If you insist, I can change it to `Puppet`.

Reply:
@mirzal yeah .. keep it `ConfigurationManagement` , also I just noticed that change to `Puppet` also does not change the component owner as well.